tests/tasks/assert_certificate_parameters.yml

```diff
@@ -1,35 +1,53 @@
 ---
-- name: Verify certificate exists in the correct location
-  file:
+- name: Retrieve certificate file stats
+  stat:
     path: "{{ cert['path'] }}"
-    state: file
   register: result
 
-- name: Verify key exists in the correct location
-  file:
+- name: Verify if certificate file exists
+  assert:
+    that:
+      - result.stat.exists == true
```

Comment:
In the similar code, I was told `== true` is redundant. ;)

Reply:
right - and to avoid the "bare variable" problem, use this: `result.stat.exists | bool`

Reply:
That's `true` :sweat_smile:
I'll get it fixed.

Reply:
Fixed in: f1bbf8139c67d180856bceb508c72ea86293e14a

Reply:
@richm I think that `result.stat.exists` is not a bare variable (that would be just `result`).